### PR TITLE
JBIDE-22083: Added Forge N&N for 4.3.1.CR1

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

**Comment by nickboldt** [comment]
Can you rename the new file to 4.3.1.Final instead of 4.3.1.CR1 ?

**Comment by gastaldi** [comment]
Done, let me know how it goes

**Comment by nickboldt** [comment]
Thanks, looks good.

File: documentation/whatsnew/forge/forge-news-4.3.1.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+= Forge Tools 4.3.1.CR1 What's New
+:page-layout: whatsnew
+:page-component_id: forge
+:page-component_version: 4.3.1.Final
+:page-product_id: jbt_core
+:page-product_version: 4.3.1.Final
+
+ifndef::finalnn[]
```
Comment (alexeykazakov) [inline documentation/whatsnew/forge/forge-news-4.3.1.Final.adoc]: So you don't want to include forge runtime update into N&N at all? This is the final N&N.

**Comment by gastaldi** [comment]
Just removed the ifndef statements. Looks like I still don't know how that exactly works. Thanks for the heads up Alexey

**Comment by alexeykazakov** [comment]
Yes this is confusing. ifndef::finalnn[] is needed when you don't want to include this section to the final N&N. If you adding something alpha/beta specific. Looks good now.

**Comment by alexeykazakov** [comment]
Squashed, rebased and pushed to master. Not in production yet.

**Comment by alexeykazakov** [comment]
Pushed to production.

**Pull request closed without merging**